modules/apps/27-interchain-accounts/controller/ibc_middleware_test.go

```diff
@@ -924,98 +904,61 @@ func (suite *InterchainAccountsTestSuite) TestOnChanUpgradeAck() {
 func (suite *InterchainAccountsTestSuite) TestOnChanUpgradeOpen() {
 	var (
 		path                *ibctesting.Path
-		isNilApp            bool
 		counterpartyVersion string
 	)
 
 	testCases := []struct {
 		name     string
 		malleate func()
-		expPanic error
+		ordering channeltypes.Order
 	}{
 		{
-			"success", func() {}, nil,
```

Comment:
the majority of these tests were testing the blocking with the middleware, since the actual ICA function does a no-op, we may be able to delete this test entirely.

Reply:
or we can simplify the test to just ensure it doesn't return an error?